nengo_dl/utils.py

```diff
@@ -364,7 +364,7 @@ class SubProgressBar(ProgressBar):  # pylint: disable=too-many-ancestors
     A progress bar representing a sub-task within an overall progress bar.
     """
 
-    def finish(self):
+    def finish(self, **kwargs):
         """Finishing a sub-progress bar doesn't start a new line."""
         super().finish(end="\r")
```

Comment:
Do you think it's better to pass `**kwargs` on to super, or just swallow them?

Reply:
I suspect it's probably better to pass them along and override only `end`, but I don't know enough about `progressbar2` to know if that could cause issues. So up to you, either works for me!

Reply:
Yeah from the error message in the original issue it looks like the `progressbar2` framework is trying to pass something to this function. Not sure what it is trying to do, but I guess might as well just pass it along and maybe it'll work properly.